src/components/v2/shared/SplitList.tsx

```diff
@@ -48,7 +48,7 @@ export default function SplitList({
             />
           </div>
         ))}
-      {ownerSplit && reservedRate ? (
+      {ownerSplit && ownerSplit.percent && reservedRate ? (
```

Comment:
This can be `ownerSplit?.percent`

Reply:
I think the better approach to this whole PR is just removing `reservedRate` from this line. Now that we're checking for having no distributions and reserved rate in the `PayoutSplitsCard` and `ReservedTokensSplitsCard` respectively, it's not necessary to have this check here. 

No need to be passing `reservedRate` into the payouts `SplitsList`

Reply:
You are right, my first commit in PR was removing `reservedRate` but i missed part that there is a check for reserved tokens in `ReservedTokensSplitsCard` and thought that needed to stay 😕 

Things should be good to merge now 👍